plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -121,16 +122,18 @@ class CodeWhispererCodeScanManager(val project: Project) {
      */
     fun isCodeScanInProgress(): Boolean = isCodeScanInProgress.get()
 
+    fun isFileScan(): Boolean = isFileScan.get()
```

Comment:
add a docstring explaining what true/false mean

Reply:
Added in commit.